docs/modules/ROOT/pages/cops_rspec_rails.adoc

```diff
@@ -33,6 +33,35 @@ end
 
 * https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/Rails/AvoidSetupHook
 
+== RSpec/Rails/HaveHttpStatus
+
+|===
+| Enabled by default | Safe | Supports autocorrection | Version Added | Version Changed
+
+| Pending
+| Yes
+| Yes (Unsafe)
+| 2.13
```

Comment:
I am not sure how this version number ended up here, but we can fix it on next release.